### ISSUE TITLE
Wrong charging_state and error in log when Skoda Enyaq is connect to charger and charger not supplying power

### ISSUE THREAD

**Issue by PerKjaer** [skodaconnect/homeassistant-myskoda#111]
**Describe the bug**
When my Skoda Enyaq is connect to the charger and the charger does not supply power, the charging_state sensor continues to have the value "connect_cable". I would expect the state to be "ready_for_charging".

At the same time I get an error in the log.

**Is this working in MySkoda app?**
N/A

**Latest working release**
N/A

**Debug logs**
```
2024-10-20 19:37:24.189 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved (None)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 258, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 453, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 168, in async_update_listeners
    update_callback()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 533, in _handle_coordinator_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1005, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1130, in _async_write_ha_state
    self.__async_calculate_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1067, in __async_calculate_state
    state = self._stringify_state(available)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1011, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 636, in state
    raise ValueError(
ValueError: Sensor sensor.charging_state provides state value 'none', which is not in the list of options provided
```

**Installation:**
 - Home Assistant variant: Docker
 - Python version (when running Core or Supervised): N/A
 - Python library versions ["python3 -m pip list"]: N/A

**Additional context**
This looks like a problem with the API, it should return "ready_for_charging" in stead of "none". 

'none' should still be treated as a valid state by the integration.

**Comment by WebSpider** [comment]
This is a duplicate of #72

**Comment by WebSpider** [comment]
At this point we do not try to interpret car states, we interpret the API results we get. If the app says the car is not plugged, we also will tell you the car is not plugged

**Comment by WebSpider** [comment]
The issue this was a duplicate of has been fixed in the recently released v1.6.0. Please give it a try!
